.github/ISSUE_TEMPLATE/feature.yaml

```diff
@@ -0,0 +1,56 @@
+name: 🚀 Feature
+description: "Submit a proposal for a new feature"
+title: "🚀 Feature: "
+labels: [feature]
```

Comment:
Can you change this to `🤔  Opportunity: Idea`

Reply:
Applied ✌️